File: Desarrollo/backend/src/main/java/com/cuidadoeterno/backend/modules/usuario/dto/CuidadorAdminResponse.java
package com.cuidadoeterno.backend.modules.usuario.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDate;
import lombok.AllArgsConstructor;
import lombok.Builder;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CuidadorAdminResponse {
    private Integer idPersona;
    private String rut;
    private String nombre;
    private String apPaterno;
    private String email;
    private String telefono;
    private String estadoVerificacion;
    private LocalDate fechaRegistro;
}

File: Desarrollo/frontend/app/src/main/java/com/cuidadoeterno/app/modules/admin/data/model/CuidadorAdminResponse.kt
package com.cuidadoeterno.app.modules.admin.data.model

data class CuidadorAdminResponse(
    val idPersona: Int,           // Antes decía idCuidador
    val rut: String,
    val nombre: String,
    val apPaterno: String?,       // Agregado del backend
    val email: String,            // Antes decía correo
    val telefono: String,
    val estadoVerificacion: String, // Antes decía estadoValidacion
    val fechaRegistro: String?      // El JSON mandará un String "YYYY-MM-DD"
)